BlockchainSdkTests/Chia/ChiaTests.swift

```diff
@@ -32,4 +38,140 @@ class ChiaTests: XCTestCase {
+            walletPublicKey: walletPublicKey
+        )
+        
+        let unspentCoins = [
+             ChiaCoin(
+                amount: 5199843583,
+                parentCoinInfo: "0x34ddaf3f1500f45b2afe2d8783f8abbde57f82be02bf2f6661095c6b20cd12cb",
+                puzzleHash: "0x9488ae2f6f0d2655aca94c6e658fdc31bd2217f74d676407112c0558d3d217d2"
+             )
+        ]
+        
+        transactionBuilder.unspentCoins = unspentCoins
+        
+        let amountToSend = Amount(with: blockchain, value: sendValue)
+        let fee = Fee(Amount(with: amountToSend, value: feeValue))
+        
+        let transactionData = Transaction(
+            amount: amountToSend,
+            fee: fee,
+            sourceAddress: sourceAddress.value,
+            destinationAddress: destinationAddress,
+            changeAddress: sourceAddress.value
+        )
+        
+        let expectedHashToSign1 = Data(hexString: "8242AB52301FF9B0DDBD5C3219729720794D96C4BDE074BEC3FB4FEFBAC5AA37831B02578BF5C3AAAF37835F9B83C29709B3DF84D9A33CCB6700CCC2B3FFF9777603CD4F12A3EE3F26C2F425B6998BA9F586E5ADF5BCBBD1935995607C9E0DC0")
+        
+        let expectedSignedTransaction = ChiaSpendBundle(
+            aggregatedSignature: "a1af85f8f921d18c1e6a81481d3e9cbf89caad7632a825dbbdeefa4c7a918903436f0b37cad1156dfd90ae94d2c0270d08e07190543dcd4e0e2a94ae2f15960ed3adb40e7ac1166bd5bfbfcc536389cfe0e1967db9ff00d09df1a1e3210460ee",
+            coinSpends: [
+                ChiaCoinSpend(
+                    coin: unspentCoins[0],
+                    puzzleReveal: "ff02ffff01ff02ffff01ff04ffff04ff04ffff04ff05ffff04ffff02ff06ffff04ff02ffff04ff0bff80808080ff80808080ff0b80ffff04ffff01ff32ff02ffff03ffff07ff0580ffff01ff0bffff0102ffff02ff06ffff04ff02ffff04ff09ff80808080ffff02ff06ffff04ff02ffff04ff0dff8080808080ffff01ff0bffff0101ff058080ff0180ff018080ffff04ffff01b08fac07255c7f3fe670e21e49cc5e70328f4181440a535cc18cf369fd280ba18fa26e28b52035717db29bff67105894b2ff018080",
+                    solution: "ffffff33ffa04475f8f79d773327466c03ce1517eeaba0ba80602ddad62e65630b311d4aeb11ff8411e1a30080ffff33ffa09488ae2f6f0d2655aca94c6e658fdc31bd2217f74d676407112c0558d3d217d2ff8501240b2c71808080"
+                )
+            ]
+        )
+        
+        let signature1 = Data(hexString: "A1AF85F8F921D18C1E6A81481D3E9CBF89CAAD7632A825DBBDEEFA4C7A918903436F0B37CAD1156DFD90AE94D2C0270D08E07190543DCD4E0E2A94AE2F15960ED3ADB40E7AC1166BD5BFBFCC536389CFE0E1967DB9FF00D09DF1A1E3210460EE")
+        
+        let buildToSignResult = try! transactionBuilder.buildForSign(transaction: transactionData)
+        let signedTransaction = try! transactionBuilder.buildToSend(signatures: [signature1])
+        
+        XCTAssertTrue(buildToSignResult.contains([expectedHashToSign1]))
+        try! XCTAssertEqual(jsonEncoder.encode(signedTransaction), jsonEncoder.encode(expectedSignedTransaction))
+    }
+    
+    @available(iOS 16.0, *)
+    func testTransactionVector2() {
```

Comment:
```suggestion
    func testTransactionVector2() throws {
```
тоже добавь throws и у всех `try!` убери плиз `!`
Будем норм хэндлить все ошибки

Reply:
Почему кстати ios16+?